scripts/azure/storage-manager.sh

```diff
@@ -0,0 +1,168 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+: "${CK8S_CONFIG_PATH:?Missing CK8S_CONFIG_PATH}"
+: "${CK8S_ENVIRONMENT_NAME:?Missing CK8S_ENVIRONMENT_NAME}"
```

Comment:
Move this down and set:
```suggestion
CK8S_ENVIRONMENT_NAME="$(yq4 .global.ck8sEnvironmentName <<< "${common_config}")"
```